style.css

```diff
@@ -49,4 +49,4 @@
 .btn-secondary {
 	border: 0.2rem solid var(--color-primary);
 	color: var(--color-primary);
-}
+} */
```

Comment:
Apart from my comments, everything looks good, and don't forget to delete the commented code. Kudos to the efforts. Well donee.